packages/core/src/sass/app/_common/layout/app-layouts.scss

```diff
@@ -185,11 +186,7 @@
         &--is-hidden {
             margin-top: 0;
         }
-        .ciq-menu {
```

Comment:
please don't remove .ciq-menu, it's a classname of smartcharts Menu element (dialog). styles might break without it.